src/maps/russia.ru.js

```diff
@@ -0,0 +1,347 @@
+export default {
+  label: 'Россия',
+  viewBox: '0 0 990 593',
```

Comment:
Could you adjust the viewBox to have no empty space (or 1px maximum) on each side of the SVG?